src/LibraryPage.tsx

```diff
@@ -114,6 +115,31 @@ const LibraryPage: React.FC = () => {
     },
     [books]
   );
+  const editBookDatabase = useCallback((book: Book) => {
+    let BookString = JSON.stringify(book);
+    fetch(`http://paralibrary.digital/api/books/${book.id}`, {
+      method: "PUT",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: BookString,
+    })
+      //The following code is used to update the page and display the edited data but currently makes the table no longer find results. Refreshing the page finds the results again
+      .then(() => {
```

Comment:
It's a good idea to check the status of the fetch before updating the books list. If there's some error and the PUT fails this might still execute giving the appearance of a successful request.

Reply:
Ah, right. Yeah, you can do something like `.then(res => { if(res.ok)...`